traj/test/test_seven_segment.py

```diff
@@ -0,0 +1,31 @@
+import nose
+import numpy as np
+
+from traj import fit_seven_segment
+
+def check_fit_seven_segment(p_start, p_end, v_max, a_max, j_max):
```

Comment:
Consider adding tests that confirm that max values are never violated.

Consider adding tests for start position is higher than end position (negative displacement).

Reply:
Good point - I'll add these in another PR. Issue created here: https://github.com/iron-ox/trajectory_smoothing/issues/5